index.js

```diff
@@ -77,7 +77,8 @@ module.exports = (app, options) => {
     const github = await app.auth()
 
     const installations = await github.paginate(
-      github.apps.listInstallations.endpoint.merge({ per_page: 100 })
+      github.apps.listInstallations({ per_page: 100 }),
```

Comment:
I'm a little confused, your change makes it work like it did with Probot 7, passing in an object as `github.apps.listInstallations.endpoint.merge({ per_page: 100 })` returns is the way moving forward